src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class Circle extends Figure {
+    private DecimalFormat decimalFormat = new DecimalFormat("#.##");

```

Comment:
If you're not providing any way to change the format, this field can be considered `final`.
Also, if you want to provide the same decimal format for every shape, you can move it to `Figure.class` and make it protected, so every subclass may access it.